src/bin/pesymbols.rs

```diff
@@ -51,10 +51,10 @@ fn download_pdb(
     println!("{}\n{}", url, user_agent);
     let mut response = Client::new()
             .get(url)
-            .header(UserAgent::new(user_agent.clone()))
+            .header(USER_AGENT, "curl/7.65.3")
```

Comment:
assuming you're passing a `&'static str` here to satisfy the type system more than to disregard the original `user_agent`, you could convert the provided `&String` to a `&str` by `&*user_agent`. That said `&String` almost always should be `&str` for API friendliness reasons.

Reply:
Very solid point @iximeow .  I am using the `user_agent` in the last commit.  How does this look?

Reply:
looks good! :+1: